3-nodes-mirror-3-dc/inventory/50-inventory.yaml

```diff
@@ -29,6 +29,12 @@ all:
             label: ydb_disk_3
         ydb_allow_format_drives: true
         ydb_skip_data_loss_confirmation_prompt: false
+        ydb_enforce_user_token_requirement: true
+        # ydb_request_client_certificate - control dynamic nodes authentication in cluster
+        # values: false - Do not require specific TLS certificate
+        #         true  - Require specific TLS certificate to join the cluster
+        # Default value: true (TODO change)
+        ydb_request_client_certificate: true
```

Comment:
The setting name is not descriptive enough: there's no way to tell that that's about registering dynamic nodes to the cluster. Or does it enforce a similar authentication method for external users as well?

Reply:
this parameter is affecting service for starting dynamic mode.
It add this parameter for ydbd if ydb_request_client_certificate is true.
--node-broker-use-tls true